src/Admin/AbstractAdmin.php

```diff
@@ -1423,8 +1423,20 @@ public function getDatagrid()
         return $this->datagrid;
     }
 
+    /**
+     * NEXT_MAJOR: Change the visibility to private and return type to void.
```

Comment:
why return type to void?

Reply:
The return value is not used in our code.

We're just calling this->buildFoo.

If you're looking at other build method we are returning void